### DISCUSSION TITLE
posts/java%E5%BC%80%E5%8F%91/%E6%85%95%E8%AF%BEoa%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/

### DISCUSSION THREAD

**Discussion by giscus** [dstweihao/dst-hugo-giscus#5] [Announcements]
# posts/java%E5%BC%80%E5%8F%91/%E6%85%95%E8%AF%BEoa%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/

慕课OA系统实战 - http://localhost:1314/posts/java%E5%BC%80%E5%8F%91/%E6%85%95%E8%AF%BEoa%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/

http://localhost:1314/posts/java%E5%BC%80%E5%8F%91/%E6%85%95%E8%AF%BEoa%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/

**Comment by dstweihao** [comment]
验证一下评论系统。
